subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/versionupgrade/CreatePullRequest.java

```diff
@@ -2,11 +2,14 @@
 
 import org.gradle.api.logging.Logger;
 import org.gradle.api.logging.Logging;
+import org.gradle.internal.impldep.org.apache.commons.lang.StringUtils;
 import org.shipkit.internal.util.GitHubApi;
 import org.shipkit.internal.util.IncubatingWarning;
 
 import java.io.IOException;
 
+import static org.gradle.internal.impldep.org.apache.commons.lang.StringUtils.isBlank;
```

Comment:
The same as with the other import - you can use Shipkit's own StringUtil.isEmpty